edge-modules/load-gen/Settings.cs

```diff
@@ -29,20 +29,31 @@ public class Settings
                     configuration.GetValue<ulong>("messageSizeInBytes", 1024),
                     configuration.GetValue("transportType", TransportType.Amqp_Tcp_Only),
                     configuration.GetValue("outputName", "output1"),
-                    configuration.GetValue("startDelay", TimeSpan.FromSeconds(2)));
+                    configuration.GetValue("testStartDelay", TimeSpan.FromMinutes(5)),
+                    configuration.GetValue("testRunDuration", TimeSpan.MaxValue),
+                    configuration.GetValue("trackingId", string.Empty));
```

Comment:
We should also update config.settings.json. Also, since the defaults are specified there, we don't really need these default values here. They do help with readability but at the cost of having to maintain in two places.

Reply:
I prefer to put defaults in the code rather than in config file.  Therefore I didn't make change in config/settings.json.

Reply:
If we are keeping the config file it might be a good idea to add the key for the new fields.

Reply:
updated.